src/localconf.js

```diff
@@ -496,11 +496,13 @@ function buildConfig(opts) {
   const cedarWrapperURL = `${hostname}cedar`;
   const gen3ZendeskURL = 'https://<SUBDOMAIN_NAME>.zendesk.com';
 
-  // Disallow gitops.json configurability of Gen3 Data Commons and CTDS logo alt text.
-  // This allows for one point-of-change in the case of future rebranding.
-  // Map href or explicit descriptor to alt text.
+  const portalLogoAltText = () => {
+    if (components?.logoAltText) return `${components.logoAltText} - home`;
```

Comment:
why append home? shouldn't alt text be completely configurable

Reply:
It appends "home" because the logo navigates back to the homepage on click, and per the ticket, we decided home should always be included to inform the user of this action.